src/test/resources/fixtures/hmh-ri/caliperEventBasicCreated.json

```diff
@@ -0,0 +1,69 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id":"urn:uuid:uniqueIdPerEvent",
+  "type": "Event",
+  "actor": {
+    "id": "https://url/to/lexileService",
+    "type": "SoftwareApplication",
+    "version": "v1"
+  },
+  "action": "Created",
+  "object": {
+    "id": "https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId/item/itemId1/attempts/completedAttemptId", (points to the same attempt in GradeEvent)
+    "type": "Attempt",
+    "assignee": {
+      "id":"https://https://www.hmhco.com/ed/users/userRefId",
+      "type": "Person",
+      "name" : "Casandra Rath"
+    },
+    "assignable": {
+      "id":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId", (points to the same assessment referenced in GradeEvent)
+      "type": "Assessment"
+    }
+  },
+  "referrer" : "https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId/attempts/completedAttemptId/scores/1", (source is from specific score for specific assessment)
+  "generated": {
+    "id": "https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId/attempts/completedAttemptId/scores/1/recommendation/1",
+    "type": "DigitalResourceCollection",
+    "name": "List of Recommended books based of lexile score for specific assessment taken by student",
+    "items": [
+      {
+        "id": "https://url/to/book/resource",
+        "type": "Document",
+        "mediaType": "Document",
+        "name": "Too Much Noise",
+        "creators": [
+          {
+            "id": "https://url/to/author/id",
+            "type": "Person",
+            "name": "Ann McGovern"
+          }
+        ],
+        "extensions": {
+          "lexile": 0
```

Comment:
@padraig-ohiceadha RI/MI have a lexile score associated with the recommended book - do you know if there is a more applicable place inside "Document" for this value otherwise is extensions ok?

Reply:
The recommendation is to add lexile levels in Caliper by adding them as learningObjectives.
I know there is a best practices document on this. I'll find it and post it to Slack...